src/components/helpers/position_hook.ts

```diff
@@ -44,16 +44,39 @@ export function useAbsolutePosition(ref: Ref): DOMCoordinates {
       return;
     }
     const { top, left } = el.getBoundingClientRect();
-    if (left !== position.x || top !== position.y) {
-      position.x = left;
-      position.y = top;
-    }
+    position.x = left;
+    position.y = top;
   }
```

Comment:
Is there a reason for this diff ? Or it's just to win two lines ?

Reply:
Yeah just win 2 lines, the `if` is useless there